lib/gcloud/jsondoc/markup_helper.rb

```diff
@@ -30,12 +30,20 @@ def linkify(name, title)
         code_obj = YARD::Registry.resolve object.namespace, name, true
         if code_obj.nil?
           name
+        elsif code_obj.type == :method
+          path = code_obj.namespace.path.gsub("::", "/").downcase
+          method_type = get_method_type code_obj
+          method_id = "#{code_obj.name}-#{method_type}"
```

Comment:
Understood. Its just that this this approach is so far removed from RDoc and YARD that we have the opportunity to improve on it.